examples/language-sdk-instrumentation/golang-push/rideshare-alloy/README.md

```diff
@@ -0,0 +1,50 @@
+# Rideshare Example with Alloy Profiling
+
+This example demonstrates how to use Alloy to receive and forward profiles from the rideshare example application.
+
+## Architecture
+
+- Regional services (us-east, eu-north, ap-south) send profiles to Alloy
+- Alloy receives profiles on port 9999 and forwards them to Pyroscope
+- Pyroscope stores and processes profiles
+- Grafana visualizes the profiling data
+
```

Comment:
@marcsanmi can we add the new image here showing how the architecture works please

Reply:
Should we wait until the new link is available? I saw the latest image update available on Monday, so I can download it and embed it, but I'd prefer to point to the public link when available. WDYT?

Reply:
https://grafana.com/media/docs/pyroscope/pyroscope_client_server_diagram_11_18_2024.png

Reply:
Done in 2723083e125cae089d3bcac34ab37cd6a2970b94